### ISSUE TITLE
Consider adding rootvalue in options

### ISSUE THREAD

**Issue by iamvdo** [robwierzbowski/node-pixrem#40]
Now, we have to use like this:

```javascript
var processed = pixrem.process(css, '20px', {browsers: 'ie >= 9'});
```

What about something like:

```javascript
var processed = pixrem.process(css, {rootValue: '20px', browsers: 'ie >= 9'});
```

Is it more consistent ? Does it provide better integration with other modules ?

**Comment by jedmao** [comment]
:+1: how else are we supposed to configure it with, say, cssnext? We need this in options!

**Comment by maoziliang** [comment]
:+1: Yes, we need this option in cssnext!

**Comment by iamvdo** [comment]
Fixed in pixrem@3.0.0
